recipe/meta.yaml

```
@@ -55,6 +55,7 @@ requirements:
     - {{ cdt('cups-devel') }}            # [linux]
     - {{ cdt('libxcb') }}                # [linux]
   host:
+    - openssl =1.1.1
```

Since this dep was missing so far, try it without the pin and then rerender.

```suggestion
    - openssl
```